nextjs/src/components/home/index.tsx

```diff
@@ -0,0 +1,63 @@
+import Link from "next/link";
+import styles from "./index.module.scss";
+import { Recommendations } from "../recommendations";
+import Image from "next/image";
+import { mockProducts } from "@/mocks/products";
+import { useEffect, useMemo } from "react";
+import {
+  trackInternalPromotionClickSpec,
+  trackInternalPromotionViewSpec,
+} from "@/lib/tracking/snowplow";
+
+const randomProduct =
+  mockProducts[Math.floor(Math.random() * mockProducts.length)];
+
+export function Home() {
+  const bannerTitle = "Shoes offers this week!";
```

Comment:
```suggestion
  const bannerTitle = "Shoe offers this week!";
```
This was annoying me